### PR TITLE
Add ErrorAction Stop to cmdlet Start-DscConfiguration - Fixes #321

### LINKED ISSUE #321: Integration tests template should use -ErrorAction Stop on Start-DscConfiguration
If `-ErrorAction Stop` is not used in integration tests for the cmdlet Start-DscConfiguration some errors does not throw resulting in passing tests (even if errors are logged).

Without `-ErrorAction Stop` (test passes); https://ci.appveyor.com/project/johlju/xsqlserver/build/6.0.991.0#L6180
With `-ErrorAction Stop` (test fails); https://ci.appveyor.com/project/johlju/xsqlserver/build/6.0.992.0#L1585

### PR DESCRIPTION
**Pull request by johlju**
- Changes to integration test template
  - Added `-ErrorAction 'Stop'` to cmdlet `Start-DscConfiguration`.

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresources/322)

### CODE REVIEW

**Comment by johlju** [comment]
@PlagueHO Do you have time to review this one?

**Comment by PlagueHO** [comment]
Of course! Will do tonight after work.

**Comment by PlagueHO** [comment]
@johlju - sorry it took so long. Reviewing now!

**Comment by PlagueHO** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[Tests.Template/integration_template.ps1, line 52 at r1](https://reviewable.io:443/reviews/powershell/dscresources/322#-Kw3zcN3ZSGAdBdAM98U:-Kw3zcN3ZSGAdBdAM98V:b-fd42xk) ([raw file](https://github.com/powershell/dscresources/blob/d20e15dabf1d428814c2734208a6386917060d9e/Tests.Template/integration_template.ps1#L52)):*
> ```PowerShell
> 
>                 $startDscConfigurationParameters = @{
>                     Path = $TestDrive
> ```

Nitpick (not blocking): Use VS Code autoformat to align = ? Not required, your call :grin:

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresources/322#-:-Kw3zq9YYJfG6pW1tSr9:bnfp4nl)*

**Comment by johlju** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[Tests.Template/integration_template.ps1, line 52 at r1](https://reviewable.io:443/reviews/powershell/dscresources/322#-Kw3zcN3ZSGAdBdAM98U:-Kw6DGm4DAnRAjy9PenU:btq1qv5) ([raw file](https://github.com/powershell/dscresources/blob/d20e15dabf1d428814c2734208a6386917060d9e/Tests.Template/integration_template.ps1#L52)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary><blockquote>

Nitpick (not blocking): Use VS Code autoformat to align = ? Not required, your call :grin:
</blockquote></details>

Done.

Ran it thru VS Code formatting using our workspace settings.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresources/322)*

**Comment by johlju** [comment]
@PlagueHO Please sign-off on the formatted file :) A few other stuff changed as well. For the better I think.

**Comment by PlagueHO** [comment]
:lgtm:

Thanks @Johlju . I didn't actually see any other changes.

@KwirkyKat , can you please merge if you're happy?

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresources/322#-:-KwElIEE-H2mrOvq1ocE:bm4xuwd)*

File: Tests.Template/integration_template.ps1 [hunk 1]
```diff
@@ -13,16 +13,16 @@
...
 
 # TODO: Customize these parameters...
-$script:DSCModuleName      = '<ModuleName>' # Example xNetworking
-$script:DSCResourceName    = '<ResourceName>' # Example MSFT_xFirewall
+$script:DSCModuleName = '<ModuleName>' # Example xNetworking
+$script:DSCResourceName = '<ResourceName>' # Example MSFT_xFirewall
 
 #region HEADER
-# Integration Test Template Version: 1.1.2
+# Integration Test Template Version: 1.1.3
```
[the remark this answers is not in the document]
Comment (johlju) [reply Tests.Template/integration_template.ps1]: Done. Forgot about that file.

**Pull request merged**